pom.xml

```diff
@@ -1149,7 +1149,7 @@
         </repository>
     </repositories>
     <properties>
-        <swagger-codegen-generators-version>1.0.28-SNAPSHOT</swagger-codegen-generators-version>
+        <swagger-codegen-generators-version>3.0.29-SNAPSHOT</swagger-codegen-generators-version>
```

Comment:
same here

Reply:
I am waiting for the review so I can update the version correctly once the https://github.com/swagger-api/swagger-codegen-generators/pull/959 is merged